docker/Dockerfile.pdo-dev

```diff
@@ -179,6 +182,25 @@ RUN wget https://downloads.sourceforge.net/project/tinyscheme/tinyscheme/tinysch
  && make FEATURES='-DUSE_DL=1 -DUSE_PLIST=1' \
  && echo "export TINY_SCHEME_SRC=$(pwd)" >> /etc/profile.d/pdo.sh
 
+# - web-assembly/wasm/wawaka
+#   - Get WAMR code
+RUN mkdir -p /project/wasm/src/
+WORKDIR /project/wasm/src
+RUN git clone https://github.com/intel/wasm-micro-runtime \
+ && cd wasm-micro-runtime \
+ && echo "export WASM_SRC=$(pwd)" >> /etc/profile.d/pdo.sh
+
+#   - get emscripten tooling
+RUN cd /project/wasm/src \
+ && git clone https://github.com/emscripten-core/emsdk.git \
+ && cd emsdk \
+ && ./emsdk install latest-fastcomp \
+ && ./emsdk activate latest-fastcomp \
+ && echo 'cd /project/wasm/src/emsdk/; if [ -z "$BASH_SOURCE" ]; then BASH_SOURCE=./emsdk_env.sh; . ./emsdk_env.sh; unset BASH_SOURCE; else . ./emsdk_env.sh; fi' >> /etc/profile.d/pdo.sh
+# Note: above convoluted BASH_SOURCE hack is necessary as (a) emsdk_env.sh 
+#   assumes we run in bash but (b) as we build we actually run in sh
+
+
```

Comment:
shouldn't there be a switch based on PDO_INTERPRETER? Why do we need to install both?

Reply:
The dev image only builds the pre-conditions and you will be able to build both versions on top of it.  Having a common base image makes things later easier.  If we ever would create a docker-compose network for a distributed "production" environment (rather than the current single EHS test environment), minimizing image size would certainly become an issue at some point in time but minimizing image size would involve also various other steps/images, so i wouldn't want to go that route until we have a requirement for it ..